src/main/kotlin/fi/metatavu/timebank/api/keycloak/KeycloakController.kt

```diff
@@ -48,6 +50,44 @@ class KeycloakController {
         }
     }
 
+    /**
+     * Gets unspentVacations attribute for Person
+     * If not set will set and return default value of -1
+     *
+     * @param user UserRepresentation
+     * @return Int unspentVacations
+     */
+    fun getUsersUnspentVacationDays(user: UserRepresentation): Int {
+        return try {
+            user.attributes["unspentVacations"]!!.first()!!.toInt()
+        } catch (e: Exception){
+            updateUsersUnspentVacationDays(user, -1)
+            -1
+        }
+    }
+
+    /**
+     * Gets spentVacations attribute for Person
+     * If not set will set and return default value of -1
+     *
+     * @param user UserRepresentation
+     * @return Int spentVacations
+     */
+    fun getUsersSpentVacationDays(user: UserRepresentation): Int {
+        return try {
+            user.attributes["spentVacations"]!!.first()!!.toInt()
+        } catch (e: Exception){
+            updateUsersSpentVacationDays(user, -1)
+            -1
+        }
+    }
+
+    fun updateAll(user: UserRepresentation, newMinimumBillable: Int, newUnspent: Int, newSpent: Int) {
```

Comment:
Missing docs.

Also I don't see where this method is being used? If it's not used, remove it.